spec/migrations/20150817213409_clear_tenant_seed_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150817213409_clear_tenant_seed")
+
+describe ClearTenantSeed do
+  migration_context :up do
+    it "works with one tenants" do
+      tenant = ClearTenantSeed::Tenant.create(:name => 'My Company')
```

Comment:
Note, migration_stub was added because many migration classes are really long names and hardcoding them in the test make the test brittle and hard to read.